crypto/src/main/scala/org/bitcoins/crypto/Sign.scala

```diff
@@ -142,10 +142,15 @@ trait Sign extends AsyncSign {
       signWithEntropy(bytes, startBytes)
     }
 
-    if (sig.bytes.length <= 70) {
-      sig
-    } else {
-      signLowR(bytes, startAt + 1)
+    // If we are using BCrypto then we can't sign with entropy
+    CryptoUtil.cryptoContext match {
```

Comment:
maybe we should file an issue for this? At least we can track it there and maybe can fix this in the future